client/src/components/SideBar/GroupSetting/index.tsx

```diff
@@ -1,29 +1,46 @@
-import React, { useState } from 'react';
-import { useSelectedGroup } from '@hooks/index';
+import React, { useEffect, useState } from 'react';
+import { useUserdata, useSelectedGroup } from '@hooks/index';
 import { ModalController } from '@customTypes/modal';
 import { GroupDeleteIcon, GroupInviteIcon } from '../../common/Icon';
 import GroupInviteModal from '../../Modal/GroupInvite';
+import GroupDeleteModal from '../../Modal/GroupDelete';
 import { GroupSettingWrapper } from './style';
 
 function GroupSetting() {
   const selectedGroup = useSelectedGroup();
+  const { userdata } = useUserdata();
+  const [selectedModal, setSelectedModal] = useState('');
+  const [isLeader, setIsLeader] = useState(false);
```

Comment:
```suggestion
  const isLeader = selectedGroup?.leader?.loginID === userdata?.loginID;
```
이거는 상수로 관리해도 될 것 같음!